apps/site/hooks/react-client/__tests__/useCopyToClipboard.test.jsx

```diff
@@ -9,10 +9,10 @@ import useCopyToClipboard from '#site/hooks/react-client/useCopyToClipboard';
 navigator.clipboard = { writeText: () => {} };
 
 await describe('useCopyToClipboard', async () => {
-  await it('should call clipboard API with `test` once', async t => {
-    t.mock.method(navigator.clipboard, 'writeText', () => Promise.resolve());
+  let TestComponent;
 
-    const TestComponent = ({ textToCopy }) => {
+  beforeEach(() => {
+    TestComponent = ({ textToCopy }) => {
       const [copied, copyText] = useCopyToClipboard();
```

Comment:
I just want to highlight the following,
```suggestion
      const [copied, copyText] = useCopyToClipboard();
```
- `copyText()` needs improvements. Should return on text is undefined or navigator do not support.
- Also copy code btn do not show notification of failed copy text. (Is it intentional?)